stripe2qbo/db/models.py

```diff
@@ -40,3 +41,23 @@ class QBOToken(Base):
     user_id: Mapped[int] = mapped_column(ForeignKey("users.id"))
 
     user = relationship("User", back_populates="qbo_token")
+
+
+class TransactionSync(Base):
+    __tablename__ = "transaction_sync"
+
+    id: Mapped[str] = mapped_column(primary_key=True)
+    created: Mapped[int] = mapped_column(nullable=False)
+    type: Mapped[str] = mapped_column(nullable=False)
+    amount: Mapped[int] = mapped_column(nullable=False)
+    description: Mapped[str] = mapped_column(nullable=False)
+    stripe_id: Mapped[str] = mapped_column(nullable=False)
+    qbo_account_id: Mapped[str] = mapped_column(nullable=False)
+    status: Optional[Literal["pending", "success", "failed"]] = None
+    # QBO ids
+    transfer_id: Mapped[str | None] = mapped_column(nullable=True)
+    invoice_id: Mapped[str | None] = mapped_column(nullable=True)
+    payment_id: Mapped[str | None] = mapped_column(nullable=True)
+    expense_id: Mapped[str | None] = mapped_column(nullable=True)
+
+    user = relationship("User", back_populates="stripe_user_id")
```

Comment:
Take a look at how QBO token does this

You should add a foreign key that references the user id.

Then add a relationship to both User and TransactionSync. `back_populates` should be the name of the field on the related model

see: https://www.google.com/search?channel=fs&client=ubuntu&q=sqlalchemy+relationship